guava/src/com/google/common/escape/ArrayBasedEscaperMap.java

```diff
@@ -65,6 +66,13 @@ private ArrayBasedEscaperMap(char[][] replacementArray) {
   // Creates a replacement array from the given map. The returned array is a
   // linear lookup table of replacement character sequences indexed by the
   // original character value.
+  @SuppressWarnings(value = {"array.length.negative", "array.access.unsafe.low",//char types are non negative: https://github.com/kelloggm/checker-framework/issues/192
+          "enhancedfor.type.incompatible"/* Key( Character type) in `map` is required to be NonNegative
```

Comment:
This is a good example of a case where you should prefix the suppression key with the checker you want to suppress - no need to be over-broad here.